commands/userargs.go

```diff
@@ -32,3 +38,16 @@ func getUserFromUserArg(c client.Client, userArg string) *model.User {
 
 	return user
 }
+
+// returns true if any of traversal patterns recognized
+func checkTraversal(arg string) bool {
+	if strings.Contains(arg, "..") {
+		return true
+	}
+
+	if strings.Contains(arg, "%2e%2e") {
+		return true
+	}
```

Comment:
Could these two be simplified with something like this?

```suggestion
	unescapedArg, _ := url.PathUnescape(arg)
	if strings.Contains(unescapedArg, "..") {
		return true
	}
```